pkg/microservice/aslan/core/environment/service/pm.go

```diff
@@ -103,7 +105,13 @@ func (p *PMService) listGroupServices(allServices []*commonmodels.ProductService
 			serviceTmpl, err := commonservice.GetServiceTemplate(
 				service.ServiceName, setting.PMDeployType, service.ProductName, "", service.Revision, p.log,
 			)
-			gp.EnvConfigs = serviceTmpl.EnvConfigs
+			if len(serviceTmpl.EnvConfigs) > 0 {
```

Comment:
`if len(serviceTmpl.EnvConfigs) > 0` could be removed.

Reply:
removed